cmd/cored/main.go

```diff
@@ -196,9 +207,9 @@ func runServer() {
 	if err != nil {
 		chainlog.Fatalkv(ctx, chainlog.KeyError, err)
 	}
-	resetInDevIfRequested(db)
+	resetInDevIfRequested(db, raftDB)
 
-	conf, err := config.Load(ctx, db)
+	conf, err := config.Load(ctx, db, raftDB)
```

Comment:
Do we know how we're going to migrate existing configurations to the new raft/protobuf? Maybe if we don't have a config, we should check the `config` table in PostgreSQL? If it's there, try to save it via raft?